app/dossierPrograms/dossierPrograms.services.js

```diff
@@ -473,3 +473,35 @@ searchModule.factory("dossiersProgramVisualizationTableFactory", [
         };
     },
 ]);
+
```

Comment:
If this code is not relevant please remove it

Reply:
Done.